lighthouse-core/test/lib/page-functions-test.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const assert = require('assert');
+const fs = require('fs');
+const jsdom = require('jsdom');
+const URL = require( '../../lib/url-shim' );
+const DOM = require('../../report/html/renderer/dom.js');
+const Util = require('../../report/html/renderer/util.js');
```

Comment:
looks like `URL` and `Util` can be removed (DOM is kind of handy to have around)